### PR TITLE
fix: Some id were localized in the templates

### PR DESCRIPTION
**Pull request by fsbraun**
## Description

This fixes issue #1335. 

A similar change has been merged into django cms core (https://github.com/django-cms/django-cms/pull/7188)

## Related resources

* #1335
* #...

## Checklist

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-filer/blob/master/docs/development.rst#contributing) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Review by vinitkumar — APPROVED** [review]
Looks good to me. Let's get this merged and released 🚢  🚀

File: filer/templates/admin/filer/folder/directory_table_list.html [hunk 1]
```diff
@@ -43,11 +43,11 @@
                             <td class="column-checkbox">
                                 {% if filer_admin_context.pick_folder and item.file_type == 'Folder' %}
                                     <a class="insertlink insertlinkButton"
-                                        href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, {{ subfolder.id }}, '{{ subfolder.quoted_logical_path }}'); return false;" >
+                                        href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, {{ subfolder.pk|safe }}, '{{ subfolder.quoted_logical_path }}'); return false;" >
```
Comment (ivarsg) [inline filer/templates/admin/filer/folder/directory_table_list.html:46]:
```suggestion
                                        href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, '{{ subfolder.pk|safe }}', '{{ subfolder.quoted_logical_path }}'); return false;" >
```

File: filer/templates/admin/filer/folder/directory_table_list.html [hunk 2]
```diff
@@ -92,7 +92,7 @@
                                     <a class="insertlink insertlinkButton" href="#" onclick="opener.dismissRelatedImageLookupPopup(window, {{ file.id|unlocalize }}, '{% file_icon_url file %}', '{{ file.label|escapejs }}'); return false;"
```
Comment (ivarsg) [inline filer/templates/admin/filer/folder/directory_table_list.html:92]:
```suggestion
                                    <a class="insertlink insertlinkButton" href="#" onclick="opener.dismissRelatedImageLookupPopup(window, '{{ file.id|unlocalize }}', '{% file_icon_url file %}', '{{ file.label|escapejs }}'); return false;"
```

File: filer/templates/admin/filer/folder/directory_thumbnail_list.html [hunk 3]
```diff
@@ -39,11 +39,11 @@
                     <div class="navigator-checkbox">
                         {% if filer_admin_context.pick_folder and item.file_type == 'Folder' %}
                             <a class="insertlink insertlinkButton"
-                                href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, {{ subfolder.id }}, '{{ subfolder.quoted_logical_path }}'); return false;" >
+                                href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, {{ subfolder.pk|safe }}, '{{ subfolder.quoted_logical_path }}'); return false;" >
```
Comment (ivarsg) [inline filer/templates/admin/filer/folder/directory_thumbnail_list.html:42]:
```suggestion
                                href="#" onclick="opener.dismissRelatedFolderLookupPopup(window, '{{ subfolder.pk|safe }}', '{{ subfolder.quoted_logical_path }}'); return false;" >
```

File: filer/templates/admin/filer/folder/directory_thumbnail_list.html [hunk 4]
```diff
@@ -85,7 +85,7 @@
                                       <a class="insertlink insertlinkButton" href="#" onclick="opener.dismissRelatedImageLookupPopup(window, {{ file.id|unlocalize }}, '{% file_icon_url file %}', '{{ file.label|escapejs }}'); return false;"
```
Comment (ivarsg) [inline filer/templates/admin/filer/folder/directory_thumbnail_list.html:85]:
```suggestion
                                      <a class="insertlink insertlinkButton" href="#" onclick="opener.dismissRelatedImageLookupPopup(window, '{{ file.id|unlocalize }}', '{% file_icon_url file %}', '{{ file.label|escapejs }}'); return false;"
```

**Review by ivarsg — COMMENTED** [review]
In order to solve issue #1335, I propose to use string literal within JavaScript code instead of integer literal. In case if PK value exceeds JavaScript `Number.MAX_SAFE_INTEGER` value, integer literal will be rounded, thus value passed to the target function will be different from the actual PK value.

**Comment by fsbraun** [comment]
@ivarsg Wow, you really push the database!

**Review by marksweb — APPROVED** [review]
Looks good to me and cleans up the JS nicely 👍

**Pull request merged**